src/utils/logger.ts

```diff
@@ -0,0 +1,27 @@
+const methods = ["log", "info", "warn", "error", "debug", "exception"] as const;
+type Methods = typeof methods[number];
+
+function consoleWrap(method: Methods) {
+  return function (...args) {
+    if (process.env.NODE_ENV !== "production") {
+      if (process.env.LOG_LEVEL === "debug") {
```

Comment:
I personally think that `NODE_ENV` is enough for now and I don't want to complicate things with `LOG_LEVEL` at this point. 

We will log on development and not in production. That should keep super simple.

Reply:
This is mainly because packages like react and styled-components have adopted this convention of only checking for NODE_ENV which I think is both sturdy and also simple at the same time. I would prefer to keep it that way.